docs/linux/zbd-api.md

```diff
@@ -113,7 +103,7 @@ below.
  *
  * start, len, capacity and wp use the regular 512 B sector unit, regardless
  * of the device logical block size. The overall structure size is 64 B to
- * match the ZBC, ZAC and ZNS defined zone descriptor and allow support for
+ * match the ZBC-, ZAC- and ZNS-defined zone descriptor and to allow support for
```

Comment:
I think we need to change "The overall structure size is 64 B to match the ZBC-, ZAC- and ZNS-defined zone descriptor and to..." to "The overall structure size is 64 B to match the size of the ZBC-, ZAC- and ZNS-defined zone descriptor and to..."